--- a/src/deque.h	2016-06-13 13:04:50.307975769 +0200
+++ b/src/deque.h	2016-06-13 13:09:08.015970878 +0200
@@ -26,9 +26,11 @@
    ( third edition, Addison-Wesley, 1997. ISBN 0-201-89683-4 )
 */
 
+/*
 #ifdef __cplusplus
 extern "C" {
 #endif
+*/
 
 #include <stddef.h>		/* size_t */
 
@@ -68,8 +70,10 @@
 
 void deque_free(struct deque_s *d);
 
+/*
 #ifdef __cplusplus
 }
 #endif
+*/
 
 #endif /* DEQUE_H */
